polkadot/runtime/parachains/Cargo.toml

```diff
@@ -49,7 +49,7 @@ frame-system = { workspace = true }
 
 xcm = { workspace = true }
 xcm-executor = { workspace = true }
-polkadot-primitives = { workspace = true }
+polkadot-primitives = { workspace = true, features=["test"] }
```

Comment:
shouldn't this be used in the dev-dependencies section below?

Reply:
yeah, somehow I swapped them.